CHANGELOG.md

```diff
@@ -1,28 +1,29 @@
+- Added a CHANGELOG file to keep better track of the codebase changes [#957](https://github.com/ucfopen/UDOIT/pull/957) (Thank you, [@dmols](https://github.com/dmols))
+- Made changes to `INSTALL_CANVAS.md` and `INSTALL.md` files to make the installation process a little easier to follow [#950](https://github.com/ucfopen/UDOIT/pull/950) (Thank you [@SimHoZebs](https://github.com/SimHoZebs))
+- Added `Makefile` to speed up development process [#958](https://github.com/ucfopen/UDOIT/pull/958) (Thank you, [@SimHoZebs](https://github.com/SimHoZebs))
 
 ### Bugfixes
 
-- Fixed issue where more than one resolved issue in the UFIXIT modal can remain. Before, the modal would only show the most recent one resolved. (Thank you, [@ssciolla](https://github.com/ssciolla))
-- Fixed case where navigating through issues on UFIXIT modal would be difficult or impossible when one is marked as fixed, since the modal will jump back to the resolved one. (Thank you, [@ssciolla](https://github.com/ssciolla))
-- Fixed issue where adding a Youtube API key to your .env file would not make UDOIT automatically consider issues revolving youtube captioning. This was resolved by allowing a 'Full Course Rescan' option in the dropdown menu of the UDOIT welcome screen (Thank you, [@taheralfayad](https://github.com/taheralfayad))
-- Upgraded php max_execution_time from 800M to 3500M, and fastcgi_read_timeout from 180 to 300, to allow for php to handle a larger amount of data, and having nginx wait longer for the processes to complete. (Thank you, [@dmols](https://github.com/dmols) and [@Thetwam](https://github.com/Thetwam))
+- Fixed issue where more than one resolved issue in the UFIXIT modal can remain. Before, the modal would only show the most recent one resolved [#892](https://github.com/ucfopen/UDOIT/pull/892) (Thank you, [@ssciolla](https://github.com/ssciolla))
+- Fixed case where navigating through issues on UFIXIT modal would be difficult or impossible when one is marked as fixed, since the modal will jump back to the resolved one [#888](https://github.com/ucfopen/UDOIT/pull/888) (Thank you, [@ssciolla](https://github.com/ssciolla))
+- Fixed issue where adding a Youtube API key to your .env file would not make UDOIT automatically consider issues revolving youtube captioning. This was resolved by allowing a 'Full Course Rescan' option in the dropdown menu of the UDOIT welcome screen [#898](https://github.com/ucfopen/UDOIT/pull/898) (Thank you, [@taheralfayad](https://github.com/taheralfayad))
+- Upgraded php `memory_limit` from 800M to 3500M, and both `max_execution_time` and `fastcgi_read_timeout` from 180 to 300, to allow for php to handle a larger amount of data, and having nginx wait longer for the processes to complete. Changes seen in [this commit](https://github.com/ucfopen/UDOIT/commit/d6c71b59dc3a353fc3d18b048473e1e09bcef423) (Thank you, [@dmols](https://github.com/dmols) and [@Thetwam](https://github.com/Thetwam))
```

Comment:
Fair point, but do you think this belongs in 'bugfixes'? Wasn't sure where it would sit; I only put it here because without it, UDOIT Admin doesn't load at all.